proiect MDPA1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+public interface Shape3d
+{
+    double SurfaceArea();
+    double Volume();
+}
+
+class Cylinder : Shape3d
+{
+    private readonly double baseRadius;
+    private readonly double height;
+
+    public Cylinder(double baseRadius, double height) =>
+      (this.baseRadius, this.height) = (baseRadius, height);
+
+    private double BaseArea() => Math.PI * Math.Pow(baseRadius, 2);
+
+    private double BaseCircumference() => 2 * Math.PI * baseRadius;
+
+    public double SurfaceArea() => 2 * BaseArea() + BaseCircumference() * height;
+
+    public double Volume() => BaseArea() * height;
+
+    public override string ToString() { return $"{GetType().Name}"; }
+}
+
+class Sphere : Shape3d
+{
+    private readonly double radius;
+
+    public Sphere(double radius) => this.radius = radius;
+
+    public double SurfaceArea() => 4 * Math.PI * Math.Pow(radius, 2);
+
+    public double Volume() => 4.0 / 3.0 * Math.PI * Math.Pow(radius, 3);
+}
+
+class CompositeShape : Shape3d

```

Comment:
Complex shape which might be out of scope of the requirements; while it's nice, it brings a lot of unneeded complexity.